gitlab/__init__.py

```diff
@@ -484,7 +484,7 @@ def delete(self, obj, id=None, **kwargs):
             params.pop(obj.idAttr)
 
         r = self._raw_delete(url, **params)
-        raise_error_from_response(r, GitlabDeleteError)
+        raise_error_from_response(r, GitlabDeleteError, expected_code=[200, 204])
```

Comment:
cf [delete return values](https://docs.gitlab.com/ce/api/snippets.html#delete-snippet), new API spec returns 204 for deletes and return no contents.